src/pages/versions/index.md

```diff
@@ -20,3 +20,4 @@
 | 2.2           | [/2.2](/2.2.x/docs/introduction/how-does-pants-work)   | [2.2.x.md](https://github.com/pantsbuild/pants/blob/main/src/python/pants/notes/2.2.x.md)   | [Blog: Pants 2.2 adds dependency inference for Protobuf](https://blog.pantsbuild.org/pants-2-2-adds-dependency-inference-for-protobuf/)                                                                                                                                                                                                            |
 | 2.1           | [/2.1](/2.1.x/docs/introduction/how-does-pants-work)   | [2.1.x.md](https://github.com/pantsbuild/pants/blob/main/src/python/pants/notes/2.1.x.rst)  | [Blog: Unlocking incremental Python 3 migrations with Pants](https://blog.pantsbuild.org/python-3-migrations/)                                                                                                                                                                                                                                     |
 | 2.0           | [/2.0](/2.0.x/docs/introduction/how-does-pants-work)   | [2.0.x.md](https://github.com/pantsbuild/pants/blob/main/src/python/pants/notes/2.0.x.rst)  | [How to upgrade from Pants 1 to Pants 2](https://www.pantsbuild.org/v1.30/docs/how-to-upgrade-to-the-v2-engine)<br/>[Blog: Introducing Pants v2](https://blog.pantsbuild.org/introducing-pants-v2/)                                                                                                                                                |
+| 1.30          | [archive](https://v1.pantsbuild.org)                   |                                                                                             |                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Want to add 1.27-1.29 as well? Or just mark this `1.x`?

Reply:
Are you suggesting one of these?

```suggestion
| 1.0 - 1.30    | [archive](https://v1.pantsbuild.org)                   |                                                                                             |                                                                                                                                                                                                                                                                                                                                                    |
```

```suggestion
| 1.x           | [archive](https://v1.pantsbuild.org)                   |                                                                                             |                                                                                                                                                                                                                                                                                                                                                    |
```

Or maybe:

```suggestion
| ≤ 1.30        | [archive](https://v1.pantsbuild.org)                   |                                                                                             |                                                                                                                                                                                                                                                                                                                                                    |
```

Reply:
Any of those would suffice for me.

Dealers choice